test/unit/utils_test.lua

```diff
@@ -34,3 +34,24 @@ function g.test_randomize_path()
 
     digest.urandom = urandom_original
 end
+
+function g.test_http_read_body()
+    local body = '--c187dde3e9318fcc6509f45f76a89424\r\n'..
+'Content-Disposition: form-data; name="file"; filename="sample.txt"\r\n' ..
```

Comment:
If I remember correctly, filename is an optional directive. Could you cover a case when there is none?

Reply:
Good point. Added.